building.md

```diff
@@ -17,13 +17,14 @@ Building above binaries require [golang](https://golang.org/dl/) with version `>
   ```
 - Verify binaries are installed
   ```sh
-  docker --help
-  # look for 
-  Management Commands:
-    generate*   Generate artifacts (CNCF Notary Project, 0.1.0)
-    notation*   Manage signatures on Docker images (CNCF Notary Project, 0.5.3-alpha)
-  
   which notation
-  # output
+  # expected output
   /home/<user>/bin/notation
   ```
+
+  If you confront `notation not found`, please add `~/bin/` to your $PATH:
+  ```sh
+  export PATH="$HOME/bin:$PATH"
+  ```
+  If you would like to add the path permanently, please add it to your `~/.profile` file.
```

Comment:
we could provide more details here, such as what content should be added into the `~/.profile` file?

Reply:
Updated.